### ISSUE TITLE
Allow Samsung Remote message on screen shows up infinite times

### ISSUE THREAD

**Issue by dreimer1986** [roberodin/ha-samsungtv-custom#10]
HASSIO: 0.100.2
Samsung custom: 3.0.3
Port: 8002
TV: UE55MU8009
Protocol used: ws

Difficult to report, but with the new version of this addin key sending same as power on/off works, but the tv shows this message three times a minute now, even when confirming it again and again.
![IMG_20191019_130531](https://user-images.githubusercontent.com/29841368/67143984-c57c6180-f271-11e9-86c8-e4a2e0fe2e68.jpg)

Only signs in log I see for now are these last two lines of it:
2019-10-19 13:05:55 INFO (SyncWorker_14) [root] Sending key KEY
2019-10-19 13:05:55 DEBUG (SyncWorker_14) [root] WS url wss://192.168.181.200:8002/api/v2/channels/samsung.remote.control?name=U2Ftc3VuZyBUViBSZW1vdGU=&token=

**Comment by djungelola** [comment]
On my TV i can choose if i need to acknowledge everytime, only one time or never. Mine is set to only the first time.

**Comment by roberodin** [comment]
Try in TV settings > network > expert > admin mobile devices > Access notifications to Disabled or First time only..

**Comment by dreimer1986** [comment]
In my case: General, External Device management, Device connection Manager, Access notifications (Loose german translation^^)
I can disable it, but it already is on First time only. My list is full of all my "Smart Device"s and a few "Home Assistant"

**Comment by xchwarze** [comment]
for WS protocol set timeout to 25 for first config

**Comment by dreimer1986** [comment]
Where? Never had a first config asking me anything. I used my original samsungtv entry, added the protocol and there was no way to set any timeout then... Where do I add that?

**Comment by xchwarze** [comment]
https://www.home-assistant.io/integrations/samsungtv/

timeout
    (time)(Optional)
    The timeout for communication with the TV in seconds.
    Default value:
    0 (no timeout)

**Comment by dreimer1986** [comment]
OK, now I feel stupid. ^^ Timeout set to 25 and it is connected and working fine now. Maybe that value should be mentioned in README or set by default I guess?

**Comment by roberodin** [comment]
@xchwarze crack!

**Comment by johntdyer** [comment]
FWIW I am still having the same issue w/ timeout set to 25, 0, or 60... I have tried all three protocols and they all yield the same results

**Comment by xchwarze** [comment]
If it takes to accept the warning it is shown several timesz on the TV
The key is to accept it the first time it appears.

**Comment by johntdyer** [comment]
i don't understand what your saying here, can you rephrase ?

**Comment by gluqui** [comment]
Hi,

I have the same problema with TV Samsung UE55MU7005. Allow Samsung remote message appears continually. I am using ws protocol and I added timeout (test 25 and 60) but it continues to appear. Sometimes it stop to appear, but if I turn off TV and turn on TV, it reappears.

Do you know how I can solve the problema, please?

Thanks

**Comment by ronsum232** [comment]
I read here and troubleshat at the same time :) Found that if you go to the connected devices manager in the TV, you'll see all those "allows" that were given. In accord with @xchwarze "The key is to accept it the first time it appears."... select all the redundant devices, delete them, and be ready to hit "allow" as soon as the delete is complete.

**Comment by wergio** [comment]
for me with model UE43RU7170U it's not working... setting "timeout: 60" gets rid of authorization pop up but integration is not working... i noticed in log he tryes to send "KEY" every 10 seconds... no response from tv...
2020-02-21 23:48:31 INFO (SyncWorker_4) [root] Sending key KEY
2020-02-21 23:48:42 INFO (SyncWorker_16) [root] Sending key KEY
2020-02-21 23:48:53 INFO (SyncWorker_10) [root] Sending key KEY
